ui/app/models/role-pki.js

```diff
@@ -104,6 +104,11 @@ export default DS.Model.extend({
   basicConstraintsValidForNonCA: attr('boolean', {
     label: 'Mark Basic Constraints valid when issuing non-CA certificates.',
   }),
+  notBeforeDuration: attr({
+    label: 'Not Before Duration',
```

Comment:
You don't have to have this line as it will "humanize" the attribute name, but fine to leave it in.